libs/infra-nest-server/src/lib/includeRawBodyMiddleware.ts

```
@@ -0,0 +1,14 @@
+import bodyParser from 'body-parser'
+
+/**
+ * Middleware that includes the raw body in the request object.
+ */
+export const includeRawBodyMiddleware = () => {
+  return bodyParser.json({
+    verify: (req: any, res, buf) => {
+      if (buf && buf.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)